src/features/transaction-wizard/data/payment-transactions.ts

```diff
@@ -0,0 +1,143 @@
+import { BuildableTransactionFormFieldType, BuildableTransaction, BuildableTransactionFormField } from '../models'
+import { zfd } from 'zod-form-data'
+import { z } from 'zod'
+import { algorandClient } from '@/features/common/data/algo-client'
+import { algos } from '@algorandfoundation/algokit-utils'
+import {
+  addressFieldSchema,
+  feeField,
+  feeFieldSchema,
+  noteField,
+  noteFieldSchema,
+  optionalAddressFieldSchema,
+  receiverField,
+  receiverFieldSchema,
+  senderField,
+  senderFieldSchema,
+  validRoundsField,
+  validRoundsFieldSchema,
+} from './common'
+import { ZERO_ADDRESS } from '@/features/common/constants'
+import { numberSchema } from '@/features/forms/data/common'
+
+const amountField = {
+  amount: {
+    label: 'Amount',
+    description: 'Amount to pay',
+    type: BuildableTransactionFormFieldType.AlgoAmount,
+    placeholder: '1',
+  } satisfies BuildableTransactionFormField,
+}
+
+const createPayment = async (data: z.infer<typeof paymentSchema>) => {
+  const transaction = await algorandClient.transactions.payment({
+    sender: data.sender,
+    receiver: data.receiver,
+    amount: algos(data.amount),
+    note: data.note,
+    ...(!data.fee.setAutomatically && data.fee.value ? { staticFee: algos(data.fee.value) } : undefined),
+    ...(!data.validRounds.setAutomatically && data.validRounds.firstValid && data.validRounds.lastValid
+      ? {
+          firstValidRound: data.validRounds.firstValid,
+          lastValidRound: data.validRounds.lastValid,
+        }
+      : undefined),
+  })
+  return transaction
+}
+
+const createAccountClose = async (data: z.infer<typeof accountCloseSchema>) => {
```

Comment:
There is definitely some opportunity to de-duplicate some common code. I decided to wait until we have more scenarios before doing this.